### PR TITLE
RD-5236 additional params for psql

### PR DESCRIPTION
**Pull request by tikhon-opsfleet**
New optional params:
- db.postgresqlSslClientVerification: true - enable or disable postgresql client certificates verification;
- tls.pgsqlSslSecretName: "pgsql-external-cert" - k8s secret name with psql ssl certs (required only if db.useExternalDB=true);
- tls.pgsqlSslCaName: "postgres_ca.crt" - subPath name for ssl CA cert in k8s secret (required only if db.useExternalDB=true);
- tls.pgsqlSslCertName: '' - subPath name for ssl cert in k8s secret (optional);
- tls.pgsqlSslKeyName: '' - subPath name for ssl key in k8s secret (optional);

### CODE REVIEW

**Review by yuvlar — APPROVED** [review]

File: cloudify-manager-worker/README.md [hunk 1]
```diff
@@ -1,28 +1,31 @@
+[//]: # -*- markdown -*-
```
Comment (yuvlar) [inline cloudify-manager-worker/README.md:1]: Is this needed?

File: cloudify-manager-worker/README.md.gotmpl [hunk 2]
```diff
@@ -0,0 +1,636 @@
+[//]: # -*- markdown -*-
```
Comment (yuvlar) [inline cloudify-manager-worker/README.md.gotmpl:1]: Is this needed?

**Review by yuvlar — CHANGES_REQUESTED** [review]
Looks great!!
Unless it's needed for those MD changes, I think the headers are a typo or smth.
If it's not needed let's omit those and merge

File: cloudify-manager-worker/README.md.gotmpl (hunk 2 above)
Comment (tikhon-opsfleet) [reply cloudify-manager-worker/README.md.gotmpl:1]: It is not necessary, just a comment for my text editor to work with this "README.md.gotmpl" file as with markdown file.
Actually I thought it shouldn't be rendered in HTML, but I was wrong, I'll change this behaviour.

Comment (tikhon-opsfleet) [reply cloudify-manager-worker/README.md.gotmpl]: Fixed, for now this comment isn't rendered in HTML.

File: cloudify-manager-worker/README.md (hunk 1 above)
Comment (tikhon-opsfleet) [reply cloudify-manager-worker/README.md]: This is the same issue as below, fixed now.

**Review by yuvlar — APPROVED** [review]

**Pull request merged**